src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -71,7 +122,88 @@ private void submitRecordDeleteOperation() {}
 
   private void submitRecordReadOperation() {}
 
-  private void submitRecordCreateOperation() {}
+  boolean completeFailedOperation(final O op, final Throwable thrown) {
+    thrown.printStackTrace();
```

Comment:
Please use the logger instead:
`LogUtil.exception(Level.DEBUG, e, "{}: operation failed: {}", stepId, op);`